src/Testcontainers.Azurite/AzuriteConfiguration.cs

```diff
@@ -7,8 +7,12 @@ public sealed class AzuriteConfiguration : ContainerConfiguration
     /// <summary>
     /// Initializes a new instance of the <see cref="AzuriteConfiguration" /> class.
     /// </summary>
-    public AzuriteConfiguration()
+    /// <param name="inMemoryPersistence">Indicates whether in-memory persistence is enabled.</param>
+    /// <param name="extentMemoryLimit">Optional memory-limit when using in-memory persistence.</param>
+    public AzuriteConfiguration(bool inMemoryPersistence = false, int? extentMemoryLimit = null)
```

Comment:
I do not think we need to keep the values. As long as we do not need them later (e.g. in the container instance) it is not necessary to keep them in the configuration instance. At least that's how we do it in other modules.